extensions/azurecore/src/extension.ts

```diff
@@ -279,3 +280,12 @@ function updatePiiLoggingLevel() {
 	const piiLogging: boolean = vscode.workspace.getConfiguration(constants.extensionConfigSectionName).get('piiLogging');
 	Logger.piiLogging = piiLogging;
 }
+
+async function getConfigTracingLevel(extensionContext: vscode.ExtensionContext): Promise<void> {
+	const configTracingLevel = 'tracingLevel';
+	const extensionConfigSectionName = 'mssql';
```

Comment:
Are you purposely tying this to the mssql logging level instead of having its own?